lmdeploy/app.py

```diff
@@ -165,5 +171,154 @@ def run(triton_server_addr: str,
     )
 
 
+def chat_stream_local(instruction: str, state_chatbot: Sequence,
+                      step: gr.State, nth_round: gr.State, model, tm_model,
+                      tokenizer, llama_chatbot):
+    """Chat with AI assistant.
+
+    Args:
+        instruction (str): user's prompt
+        state_chatbot (Sequence): the chatting history
+        llama_chatbot (Chatbot): the instance of a chatbot
+        model_name (str): the name of deployed model
+    """
+    seed = random.getrandbits(64)
+    bot_summarized_response = ''
+    state_chatbot = state_chatbot + [(instruction, None)]
+    instruction = model.get_prompt(instruction, nth_round == 1)
+    if step >= tm_model.session_len:
+        print('WARNING: exceed session max length.'
```

Comment:
Can this warning be shown on UI?
btw, if it exceeds the limits, server should reject the request